### PR TITLE
Secret Level pack

### PR DESCRIPTION
**Pull request by DarkVexon**
Adds a new pack with some meta elements that focuses around Weak and Vulnerable as well as conditional effects.

Since the pack interacts with cards that glow gold to signify a special trigger, I did a pass on those in other packs; there weren't too many that needed it.

I also fixed one or two missing periods and other small grammatical things.

Also made some balance changes to my existing content:
Beacon+: Retain instead of 2 cards
Eight Ball: +2 damage
Final Form: Only works for 5 turns :(
Last Resort: Grants +1 Block, and energy next turn if used but not exhaust
Midnight Strike: Upgrade nerfed ish, now +3 damage and 48(60) proc damage but no 0 cost upgrade
Struck by a Train: -2 base damage and scaling, -1 base damage and scaling on upgrade
Super Nova: -7 base HP loss
Trash: Always exhausts at random
Reverberation: -2 Block

Also, one of my cards in the Oddities pack (Beacon) was almost going to be named Barrier Beacon, and then Jedi later by complete coincidence made a card named Barrier Beacon that used the version of the art that I left in accidentally, so I swapped it out with a new art so 2 cards don't have the same art.

### CODE REVIEW

File: src/main/java/thePackmaster/patches/secretlevelpack/SpecialCardGlowCheckPatch.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            for (AbstractPower q : AbstractDungeon.player.powers) {
+                if (q.ID.equals(AchievementHunterPower.POWER_ID)) {
+                    q.onSpecificTrigger();
+                }
+            }
+        }
+    }
+
+    private static boolean equalsIgnoreA(Color test, Color other) {
+        return test.r == other.r && test.b == other.b && test.g == other.g;
```
Comment (erasels) [inline src/main/java/thePackmaster/patches/secretlevelpack/SpecialCardGlowCheckPatch.java:35]: no null safety

File: src/main/java/thePackmaster/powers/odditiespack/FinalFormPower.java [hunk 2]
```diff
@@ -43,14 +45,13 @@ public void update() {
...
-
-
+        atb(new ReducePowerAction(owner, owner, this, 1));
     }
 
     @Override
     public void updateDescription() {
         if (card != null)
-            description = DESCRIPTIONS[0] + FontHelper.colorString(card.name, "y") + DESCRIPTIONS[1];
+            description = DESCRIPTIONS[0] + amount + (amount == 1 ? DESCRIPTIONS[1] : DESCRIPTIONS[2]) + DESCRIPTIONS[3] + FontHelper.colorString(card.name, "y") + DESCRIPTIONS[4];
```
Comment (erasels) [inline src/main/java/thePackmaster/powers/odditiespack/FinalFormPower.java:54]: make sure that zhs has these

File: src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Cardstrings.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "${ModID}:Joust": {
+    "NAME": "Joust",
+    "DESCRIPTION": "Can only target Weak enemies. NL Deal !D! damage. NL Apply !M! Weak.",
+    "EXTENDED_DESCRIPTION": [
+      "That foe is not Weak!"
+    ]
+  },
+  "${ModID}:Conglaturation": {
+    "NAME": "Conglaturation!",
+    "DESCRIPTION": "Deal !D! damage. NL If you played a glowing card this turn, gain [E] and draw 1 card."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Cardstrings.json:27]: blue is also a glow

File: src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Keywordstrings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Keywordstrings.json:1]: can be removed

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/java/thePackmaster/powers/odditiespack/FinalFormPower.java (hunk 2 above)
Comment (DarkVexon) [reply src/main/java/thePackmaster/powers/odditiespack/FinalFormPower.java:54]: Just adding the ENG version of the strings now for crash safety, I don't want to step on mwalls' work

File: src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Cardstrings.json (hunk 3 above)
Comment (DarkVexon) [reply src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Cardstrings.json]: Clarified; changed those cases to "gold-glowing"

**Pull request merged**